distributed/versions.py

```diff
@@ -39,18 +38,18 @@
 }
 
 
-def get_versions(
-    packages: Iterable[str | tuple[str, Callable[[ModuleType], str | None]]]
-    | None = None
-) -> dict[str, dict[str, Any]]:
+def get_versions(packages: Iterable[str] | None = None) -> dict[str, dict[str, Any]]:
```

Comment:
```suggestion
def get_versions(packages: Iterable[str] = ()) -> dict[str, dict[str, Any]]:
```

Reply:
I kept support for None because `versions` on the base `ServerNode` uses it as the default argument. Wasn't sure if not supporting None in `Client.get_versions` or any other use of this would be considered a breaking change or not.
https://github.com/dask/distributed/blob/573128245ccd8da2f6a1d4e237a9611294745760/distributed/node.py#L28-L29

I'd also prefer the empty tuple default if you think that's ok.

Reply:
Yes the caller in node.py should be changed too. 
client.py already defaults to an empty list and not none and none is undocumented, so it's not a breaking change.